epub33/rs/index.html

```diff
@@ -336,27 +336,21 @@ <h2>Internationalization</h2>
 			<section id="sec-epub-rs-network-access">
 				<h3>Network Access</h3>
 
-				<p>Reading Systems may support network access to enable the <a href="#sec-epub-rs-conf-remote-res"
-						>retrieval of remote resources</a> and to allow <a>Scripted Content Documents</a>
-					<a href="#confreq-rs-scripted-limit">to communicate with web-hosted APIs and retrieve
+				<p>Reading Systems may support network access to <a href="#sec-epub-rs-conf-remote-res"
+						>retrieve remote resources</a> and to allow <a>Scripted Content Documents</a> to
+					<a href="#confreq-rs-scripted-limit">communicate with web-hosted APIs and retrieve
 					resources</a>.</p>
 
 				<p>Providing network access, however, increases both the security risks to the Reading Systems and the
 					security and privacy risks to users. These risks are often unique to Reading Systems and the
-					platforms they run on &#8212; the browser cores most dedicated Reading System apps are built on do
-					not offer the same security and privacy controls as web browsers themselves. Consequently,
-					developers need to use extra caution when allowing network access, and more thoroughly test that
-					their Reading Systems are not vulnerable to attacks. More information about these risks is provided
-					in <a href="#sec-security-privacy"></a>.</p>
-
-				<p>If Reading System developers opt to allow network access, it is strongly RECOMMENDED that they
-					include methods to notify the user that network activity is occurring and/or that allow them to
-					disable it.</p>
-
-				<p>For example, users might be presented with a global setting that disables all network access on a
-					Reading System-wide or publication-by-publication basis (i.e., no downloading of remote resources or
-					scripting access). Users could also be provided the option to block or terminate network access
-					occurring within a publication to prevent unwanted scripting interactions.</p>
+					platforms they run on &#8212; the browser cores that most Reading Systems are built on do not offer
+					the same security and privacy controls as web browsers themselves. Consequently, developers need to
+					use extra caution when allowing network access, and more thoroughly test that their Reading Systems
+					are not vulnerable to attacks. More information about these risks is provided in
+					<a href="#sec-security-privacy"></a>.</p>
+
+				<p>If Reading System developers allow network access, it is strongly RECOMMENDED that they notify the
+					user when network activity occurs and/or allow them to prevent it.</p>
```

Comment:
Every time I read the "and/or" it sounds like it could mean to prevent notifications.

Maybe split this in two, so more like: 

> If Reading System developers allow network access, it is strongly RECOMMENDED both that they:
> - notify users when network activity is occurring; and
> - provide users options to block access to the network (e.g., an option to disable access to all a user's EPUB Publications, or an option to temporarily disable access from within an EPUB Publication).

The last para is bloated, but some sense of what we mean by blocking access wouldn't hurt. But feel free to tweak the suggestion.

Reply:
Sure, how's this?